tests/spread/integration/apache2/task.yaml

```diff
@@ -0,0 +1,26 @@
+summary: Integration tests for apache2
+
+execute: |
+  rootfs="$(install-slices apache2_bins base-passwd_data)"
+
+  #sed -i '1a set -x' "${rootfs}/usr/sbin/apache2ctl"
+  mkdir -p "${rootfs}/dev"
+  mount --rbind /dev "${rootfs}/dev"
+  echo -e "ServerName localhost" >> "${rootfs}/etc/apache2/apache2.conf"
+  test "$(chroot "${rootfs}" a2query -M)" = "prefork"
+  test "$(chroot "${rootfs}" a2dismod mpm_event | tail -n -1)" = "Module mpm_event already disabled"
+  test "$(chroot "${rootfs}" a2enmod mpm_prefork | tail -n -1)" = "Module mpm_prefork already enabled"
+  test "$(chroot "${rootfs}" a2dissite 000-default | head -n 1)" = "Site 000-default disabled."
+  test "$(chroot "${rootfs}" a2ensite 000-default | head -n 1)" = "Enabling site 000-default."
+  test "$(chroot "${rootfs}" a2disconf charset | head -n 1)" = "Conf charset disabled."
+  test "$(chroot "${rootfs}" a2enconf charset | head -n 1)" = "Enabling conf charset."
+  mkdir -p "${rootfs}/var/www/html"
+  echo "Hello, Apache is working!" > "${rootfs}/var/www/html/index.html"
+  chmod 755 "${rootfs}/var/www"
+  chmod 755 "${rootfs}/var/www/html"
+  chmod 644 "${rootfs}/var/www/html/index.html"
+  chmod 755 ${rootfs}
+  chroot "${rootfs}" apache2ctl start
+  cat "${rootfs}/var/log/apache2/error.log"
+  cat "${rootfs}/var/log/apache2/access.log"
+  test "$(curl -s -o /dev/null -w "%{http_code}\n" 127.0.0.1)" = "200"
```

Comment:
I think we need to add an unmount of `"${rootfs}/dev"` at the end?
Something like this?
```shell
umount -l "${rootfs}/dev"
```